### PR TITLE
UIEH-908 Replace Choose custom package component

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
[UIEH-908](https://issues.folio.org/browse/UIEH-908) Replace Choose custom package component
 
Requirement: On Create/Edit Custom Resource page AND the Title record on the Choose custom package modal replace the 
`Select` component with `Selection` component

### CODE REVIEW

File: src/components/title/_fields/package-select/package-select-field.js [hunk 1]
```diff
@@ -14,29 +13,22 @@ function PackageSelectField({ options }) {
     <div data-test-eholdings-package-select-field>
       <Field
         name="packageId"
-        component={Select}
+        component={Selection}
         label={<FormattedMessage id="ui-eholdings.label.package" />}
         validate={validate}
         onBlur={null} // preventing validation that is in onBlur
+        placeholder={<FormattedMessage id="ui-eholdings.title.chooseAPackage" />}
+        dataOptions={options.slice(0, 100)}
```
Comment (BogdanDenis) [inline src/components/title/_fields/package-select/package-select-field.js:21]: Please remove `slice` here so we don't forget about it when back-end is updated

File: src/components/title/_fields/package-select/package-select-field.js [hunk 2]
```diff
@@ -14,29 +13,22 @@ function PackageSelectField({ options }) {
...
-            )}
-          </FormattedMessage>
-        ) : (
-          <FormattedMessage id="ui-eholdings.search.loading">
-            {option => (
-              <option value="" disabled>{option}</option>
-            )}
-          </FormattedMessage>
-        )}
-        {options.filter(option => option.label !== '').map(({ disabled, label, value }) => (
```
Comment (BogdanDenis) [inline src/components/title/_fields/package-select/package-select-field.js:36]: Please use `formatter` property of `Selection` component to support disabling options
https://ux.folio.org/storybook/?path=/story/selection--basic-usage

File: test/bigtest/tests/title-create-test.js [hunk 3]
```diff
@@ -69,12 +69,11 @@ describe('TitleCreate', () => {
...
     });
 
-    describe('creating a new title', () => {
+    describe.only('creating a new title', () => {
       beforeEach(() => {
-        return TitleCreatePage
-          .fillName('My Title')
-          .selectPackage(packages[0].name)
-          .save();
+        TitleCreatePage.fillName('My Title');
```
Comment (BogdanDenis) [inline test/bigtest/tests/title-create-test.js:74]: Have you tried turning this `beforeEach` into an async function and awaiting these calls?

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: test/bigtest/tests/title-create-test.js (hunk 3 above)
Comment (yelyzavetakhokhlova) [reply test/bigtest/tests/title-create-test.js:74]: Yeah, didn't work either

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**